api/src/queries/project-queries.ts

```diff
@@ -459,3 +461,52 @@ function generateGeometryCollectionSQL(geometry: Feature[]): SQLStatement {
 
   return sqlStatement;
 }
+
+/**
+ * SQL query to insert a project stakeholder partnership row.
```

Comment:
question: is this comment valid? is it inserting a stakeholder partnership row or a permit row? or are those the same